src/actions/guides/testing/introduction.cr

```diff
@@ -0,0 +1,75 @@
+class Guides::Testing::Introduction < GuideAction
+  guide_route "/testing/introduction"
+
+  def self.title
+    "Intro to Testing"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Getting Started
+
+    All of your tests will go in the `spec/` directory located at the top level of your application.
+    When you generate your Lucky project, there are a few files generated for you to help make testing
+    easier.
+
+    > The files generated will be different based on Full/Api and/or Authentication generated.
+
+    ### spec_helper.cr
+
+    The `spec/spec_helper.cr` file will require all of the necessary files and configuration your testing
+    suite will need to run. Every time you add a new spec file for testing, you will need to require this file
+    first. Read more on [testing in Crystal](https://crystal-lang.org/reference/guides/testing.html).
+
+    ## Flows
+
+    If you need to test your HTML views and/or the interactivity of your site (e.g. a user clicking a button to login),
+    you will create your flow specs in this directory.
+
+    > If you generated authentication for your app, you will have a few flow specs added in for you!
+
+    ## Setup
+
+    This directory is used to setup all of your testing. Any additional setups needed before your spec suite runs
+    should go in here. They will be automatically required and ran in alphabetical order.
+
+    ### setup/clean_database.cr
+
+    Before every spec, your database will automatically be truncated wiping out any test records you've added.
+
+    ### setup/reset_emails.cr
+
+    Every test email sent is tracked as delivered. This will clear out "delivered" test email before each spec.
+
+    ### setup/setup_database.cr
+
+    This will create and migrate your test database for you. By default, Lucky apps will use the name of your app
+    with the current environment as a suffix. (i.e. my_app_test or my_app_development).
+
+    This is ran for you when you run `crystal spec`.
```

Comment:
I thought the same thing originally, but I wanted to stress the point that you don't have to run it because people don't really "read" `%s/people/I/` 😅